src/abstractdataframe/io.jl

```diff
@@ -417,7 +423,12 @@ function printtable(io::IO,
             elseif isnothing(df[i, j])
                 print(io, nothingstring)
             else
-                if ! (etypes[j] <: Real)
+                if df[i,j] isa Markdown.MD
+                    print(io, quotemark)
+                    r=repr(df[i,j])
+                    escapedprint(io, first(r,max(0,length(r)-1)), quotestr)
```

Comment:
I guess this is what you want to do here - right?
```suggestion
                    escapedprint(io, chomp(r), quotestr)
```